test/system/500-networking.bats

```diff
@@ -317,14 +317,16 @@ load helpers.network
     run curl -s -S $SERVER/index.txt
     is "$output" "$random_1" "curl 127.0.0.1:/index.txt"
 
-    # rootless cannot modify iptables
+    # rootless cannot modify the host firewall
     if ! is_rootless; then
-        # flush the port forwarding iptable rule here
-        chain="CNI-HOSTPORT-DNAT"
-        if is_netavark; then
-            chain="NETAVARK-HOSTPORT-DNAT"
-        fi
-        run iptables -t nat -F "$chain"
+        # for debugging only
+        iptables -t nat -nvL || true
+        nft list ruleset     || true
```

Comment:
Will a future maintainer, looking at failure logs, know what these are? I know it's duplication, but I like to include the commands in the output:
```bash
    echo "$_LOG_PROMPT iptables -t nat -nvL"
    iptables -t nat -nvL || true
    echo "$_LOG_PROMPT nft list ruleset"
    nft list ruleset
```

Reply:
yeah I can add this although format is very unique for both so it is easy to know what they are, but I guess I can say this because I look at this stuff often. I will include if I have to repush again.